src/pcre2_context.c

```diff
@@ -409,6 +410,29 @@ ccontext->stack_guard_data = user_data;
 return 0;
 }
 
+PCRE2_EXP_DEFN int PCRE2_CALL_CONVENTION
+pcre2_set_optimize(pcre2_compile_context *ccontext, uint32_t directive)
+{
+if (directive == PCRE2_OPTIMIZATION_NONE)
+  {
```

Comment:
Don't need braces for single statements.

Reply:
That's true, but a different branch in the same if-else chain has multiple statements.

I would advise against omitting the braces in one branch and including them in another.

Reply:
which is why it would be better to use switch/case